### PR TITLE
Gobierto Dashboards / Create a pagination component

### LINKED ISSUE #3054: Contracts / Pagination
We need to add a pagination to contracts / tenders list.

Add a previous and next buttons

There are no design and we don't have any Vue component for this (in investments we have a view all button).

### PR DESCRIPTION
**Pull request by jorgeatgu**
Closes #3054

## :v: What does this PR do?

Create a pagination component for our `Vue-components` library. Include pagination in the Gobierto Dashboards Contracts

## :mag: How should this be manually tested?

[subsidies](https://getafe.gobify.net/dashboards/subvenciones/)
[contracts](https://getafe.gobify.net/dashboards/contratos/)

### After this PR

pagination

## :book: Does this PR require updating the documentation?

- [x] Add documentation

### CODE REVIEW

**Comment by ferblape** [comment]
@jorgeatgu can you review the conflicts?

File: app/javascript/gobierto_dashboards/webapp/components/Table.vue [hunk 1]
```diff
@@ -12,14 +12,19 @@
...
         :key="item.id"
         :item="item"
         :routing-member="routingMember"
         :routing-attribute="routingAttribute"
         :columns="columns"
       />
     </table>
+    <Pagination
+      :data="items"
+      :items-per-page="10"
```
Comment (ferblape) [inline app/javascript/gobierto_dashboards/webapp/components/Table.vue]: 15 items per page

**Review by ferblape — CHANGES_REQUESTED** [review]
Maybe hide the paginator when it's not necessary?

Screenshot 2020-07-28 at 06 50 34

File: app/javascript/lib/vue-components/modules/Pagination.vue [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          :disabled="pageNumber === page"
+          @click="page = pageNumber"
+        >
+          {{ pageNumber }}
+        </button>
+      </li>
+    </ul>
+    <button
+      class="gobierto-pagination-btn"
+      :disabled="page === pages.slice(-1)[0]"
```
Comment (Crashillo) [inline app/javascript/lib/vue-components/modules/Pagination.vue]: Computed and comment (right part)

**Review by Crashillo — APPROVED** [review]

**Comment by ferblape** [comment]
Ups, another detail from my small screen: could you scroll to top after changing the page?

Screenshot 2020-07-28 at 11 33 47

**Comment by jorgeatgu** [comment]
> Ups, another detail from my small screen: could you scroll to top after changing the page?
> 
> Screenshot 2020-07-28 at 11 33 47

Done!

scroll_pagination

**Pull request merged**